### PR TITLE
AWS SIG Proposal

### PR DESCRIPTION
**Pull request by imosquera**

### CODE REVIEW

File: sig-aws/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Roles & Responsibilities of the SIG Leads 
+
+### SIG Lead(s):
+
+**Clare Liguori** \
+Principal Software Engineer \
+_AWS_
+
+**Rob Zeinert** \
```
Comment (robzienert) [inline sig-aws/README.md]:
```suggestion
**Rob Zienert** \
```

File: sig-aws/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Proposal: Spinnaker AWS Special Interest Group
+
+## Roles & Responsibilities of the SIG Leads 
+
+### SIG Lead(s):
+
+**Clare Liguori** \
```
Comment (robzienert) [inline sig-aws/README.md]: We should link these names to our respective github profiles.

**Comment by aglover** [comment]
FYI @spinnaker/sc

File: sig-aws/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    *   CodeCommit for source
+    *   CodeBuild for build / bake
+    *   ECR for Docker registry
+    *   CloudWatch for canary analysis (Kayenta)
+*   Support users on their issues and feature requests
+*   Documentation for all things AWS for Spinnaker
+*   Consult with other SIGs and the community on how to apply mechanisms owned by SIG AWS. Examples include:
+    *   Review escalation implications of feature and API designs as it relates to core Spinnaker components (orca, clouddriver, echo, deck, etc)
+    *   AWS Clouddriver implementation and interface design
+    *   Implementing and hardening tests, scale tests and documentation
```
Comment (pstout) [inline sig-aws/README.md]: As I read the document, this seems like something I would see as the responsibility of the core Spinnaker developers (TOC?).  Are you trying to make this point about the AWS specific code?

File: sig-aws/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+AWS SIG also acts as a forum for discussions and feature demos for Spinnaker users on AWS users/developers to raise their feature requests and support issues. SIG leads in collaboration with SIG members will develop, maintain and prioritize a backlog of features.  This prioritization will be published as the AWS SIG Roadmap.
+
+
+## Scope of Work
+
+Spinnaker integrations specific to AWS including:
+
+*   Integrations, interfaces, libraries and extension points for all low-level AWS modules such as IAM, storage, networking, load balancers, registry, security, monitoring/logging at the instance or container level
+*   Development of Spinnaker APIs to provide software delivery best practices to computational AWS services which might include, but not limited to, the following:
```
Comment (pstout) [inline sig-aws/README.md]:
```suggestion
*   Development of Spinnaker APIs to provide software delivery best practices for AWS computational services which might include, but are not limited to, the following:
```

File: sig-aws/README.md (hunk 3 above)
Comment (imosquera) [reply sig-aws/README.md]: @pstout 
Correct.  All of these would be about how AWS implementations might run into cross-cutting concerns and part of other systems like testing and performance that might be affected by what we're developing in the AWS modules.  Open to changing the language or omitting the section in its entirety.

File: sig-aws/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+AWS SIG also acts as a forum for discussions and feature demos for Spinnaker users on AWS users/developers to raise their feature requests and support issues. SIG leads in collaboration with SIG members will develop, maintain and prioritize a backlog of features.  This prioritization will be published as the AWS SIG Roadmap.
+
+
+## Scope of Work
+
+Spinnaker integrations specific to AWS including:
+
+*   Integrations, interfaces, libraries and extension points for all low-level AWS modules such as IAM, storage, networking, load balancers, registry, security, monitoring/logging at the instance or container level
+*   Development of Spinnaker APIs to provide software delivery best practices for AWS computational services which might include, but not limited to, the following:
```
Comment (clareliguori) [inline sig-aws/README.md]: Nit: we typically refer to "AWS compute services", not computational: https://docs.aws.amazon.com/whitepapers/latest/aws-overview/compute-services.html

**Comment by aglover** [comment]
Seems like all of the suggestions and feedback have been incorporated into this proposal. 

My vote is a +1.

File: sig-aws/README.md (hunk 3 above)
Comment (pstout) [reply sig-aws/README.md]: @imosquera How about something like:
```
Implementing and improving core functionality (such as tests and documentation) to clarify and validate the contract between the core and and the AWS specific functionality
```

File: sig-aws/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+AWS SIG also acts as a forum for discussions and feature demos for Spinnaker users on AWS users/developers to raise their feature requests and support issues. SIG leads in collaboration with SIG members will develop, maintain and prioritize a backlog of features.  This prioritization will be published as the AWS SIG Roadmap.
+
+
+## Scope of Work
+
+Spinnaker integrations specific to AWS including:
+
+*   Integrations, interfaces, libraries and extension points for all low-level AWS modules such as IAM, storage, networking, load balancers, registry, security, monitoring/logging at the instance or container level
+*   Development of Spinnaker APIs to provide software delivery best practices for AWS compute services which might include, but not limited to, the following:
```
Comment (pstout) [inline sig-aws/README.md]:
```suggestion
*   Development of Spinnaker APIs to provide software delivery best practices for AWS compute services which might include, but are not limited to, the following:
```

**Comment by pstout** [comment]
While there are two minor editorial suggestions from me outstanding that I think would clarify the vision, I am fine with the proposal.

My vote is also +1.

**Comment by skim1420** [comment]
+1 thank you.

**Comment by duftler** [comment]
+1

**Comment by rmeshenberg** [comment]
+1

**Comment by aglover** [comment]
@imosquera can you address @pstout comments? I can merge this ASAP.

File: sig-aws/README.md (hunk 4 above)
Comment (imosquera) [reply sig-aws/README.md]: i've updated with this language

File: sig-aws/README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+1. Coordinate communication.
+2. Set meeting dates and objectives for the coming year.
+3. Curate agenda for each meeting.
+4. Raise awareness and invite appropriate participants.
+5. Encourage and foster an inclusive environment.
+6. Facilitate the identification and development of resources (people or funding) for support of AWS within Spinnaker.
+
+
+### Meeting Cadence
+1 hour, monthly, on Google Hangouts.
```
[the remark this answers is not in the document]
Comment (imosquera) [reply sig-aws/README.md]: i've updated

**Pull request merged**